### DISCUSSION TITLE
Why public and restricted orders have order type 2?

### DISCUSSION THREAD

**Discussion by mohoff** [ProjectOpenSea/seaport-js#66] [Q&A] [answered]
Hi, no matter how I list NFTs for sale, as private or public orders, the Seaport order type is always 2 on https://testnets.opensea.io, while I expected it to be 0 (public) and 2 (private). At least that's my interpretation of https://github.com/ProjectOpenSea/seaport-js/blob/16c77475a6d61452eae01a3092f2e0c435dfc76d/src/constants.ts#L40_L45.

Can someone explain why it is 2 in both cases and under what circumstances an order type of 0 applies? Thanks!

**Comment by ryanio** [comment, marked as answer]
The difference between full and restricted is that restricted additionally asks the zone for isValidOrder, and allows the zone to e.g. complete an auction. So opensea uses their zone with restricted order type. You are free to create full (non-restricted) orders and execute them on seaport independently.

**Reply by mohoff** [reply]
Thanks @ryanio, I'm trying to understand this better so following up:

> So opensea uses their zone with restricted order type

Is that the zero-hash zone at contract https://etherscan.io/address/0x004C00500000aD104D7DBd00e3ae0A5C00560C00 ? It just returns the magic value in `isValidOrder` so what's the benefit of using this zone (restricted order, order type 2) vs no zone (full order, order type 0)?

> You are free to create full (non-restricted) orders

How's that possible through the OpenSea UI? I can only pass a specific taker address

**Reply by ryanio** [reply]
That is the global pausable zone, a safe measure in case there is an issue with Seaport then opensea orders can be paused until it is resolved. It is what is used through the opensea ui, you cannot opt out currently. I just meant you aren't forced to use it if you'd like to interact with the base seaport contract.

Check out the zone documentation in the zones 1.1 Pr for more info about zones.

**Reply by mohoff** [reply]
Thanks. That must be https://github.com/ProjectOpenSea/seaport/pull/515

Lastly, @ryanio can you please clarify the rollout/status of Zones 1.1? The PR is still open but the OpenSea default pausable zone seems to be deployed and live already.

**Reply by ryanio** [reply]
@mohoff you can see the deployed zone contract that is in use here https://etherscan.io/address/0x004C00500000aD104D7DBd00e3ae0A5C00560C00#contracts I believe the PR has some small updates that will be deployed in the future, but the core logic remains the same
